doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -187,7 +187,7 @@ async def build(self, metadata: ImageMetadata):
                 if self._config.dry_run:
                     logger.info("Dry run: Would have inserted build record in Konflux DB")
                 else:
-                    await self.update_konflux_db(metadata, build_repo, pipelinerun, outcome, building_arches, pod_list)
+                    _, outcome = await self.update_konflux_db(metadata, build_repo, pipelinerun, outcome, building_arches, pod_list)
```

Comment:
I don't think we should have the gating as part of the update_konflux_db func. It would be better to do it after line 185, i.e. when build has finished.

Reply:
makes sense - updated in https://github.com/openshift-eng/art-tools/pull/1570/commits/e7f988d9c6288eea1bebb7772a03797f100cead3